### ISSUE TITLE
Blank lines inserted before private imports

### ISSUE THREAD

**Issue by Barugon** [dlang-community/dfmt#301]
Before dfmt:
```D
private import std.file;
private import std.path;
import std.stdio;
private import std.string;
```

After dfmt:
```D
private import std.file;

private import std.path;
import std.stdio;

private import std.string;
```

**Comment by stefan-koch-sociomantic** [comment]
@Barugon what would be your expectation for imports to be formatted ?

**Comment by Barugon** [comment]
If they're not private then a blank line is not inserted.

Before dfmt:
```D
import std.file;
import std.path;
import std.stdio;
import std.string;
```

After dfmt:
```D
import std.file;
import std.path;
import std.stdio;
import std.string;
```

I would expect the behavior to be the same for private imports. If I make them all private then this is what I get after using dfmt:
```D
private import std.file;

private import std.path;

private import std.stdio;

private import std.string;
```

**Comment by stefan-koch-sociomantic** [comment]
I have now run into this as well.
I am going to fix this; however it's not clear to me why this actually happens ;)

**Comment by stefan-koch-sociomantic** [comment]
This is closed via #305
